backend/routes/ckan/organizations.js

```diff
@@ -61,7 +61,7 @@ var addRoutes = function(router){
         let orgTTL = 86400;
     
         //let keys = Object.keys(req.query);
-        let reqUrl = url + "/api/3/action/organization_list_related?all_fields=True";
+        let reqUrl = url + "/api/3/action/organization_list_related";
```

Comment:
Initially if we didn't have all fields true we didn't get all the fields we needed I assume this has been tested/addressed with back end changes

Reply:
I belive it is now being handled by changes in the backend here:
https://github.com/bcgov/ckanext-bcgov/pull/895/files#diff-78cd99d032813bd0bb783da697a4dc8d25473d9377ba50f205393b034b688849